### ISSUE TITLE
validators breaks autocompleteItems

### ISSUE THREAD

**Issue by clement911** [Gbuomprisco/ngx-chips#738]
**I'm submitting a ...**
```
[x] bug report
```

I have a tag-input with a autocompleteiItems and validators.
When I click an item form the auto complete list, the list closes but the item is not added.
If I remove the validators, it gets added as expected. 
Note that the autocomplete items are valid.

**Angular version:** 5.2.0
**ngx-chips version:** 1.8.0
**Browser:** Chrome

```
<tag-input [(ngModel)]='selectedRecipients'
                       style="display: inline-block;"
                       [secondaryPlaceholder]="'Add email'"
                       [placeholder]="'Add email'"
                       [separatorKeys]="[',']"
                       [errorMessages]="errorMessages"
                       [validators]="validators"
                       [addOnBlur]="true"
                       [modelAsStrings]="true">
                <tag-input-dropdown [autocompleteItems]="existingRecipients">
                </tag-input-dropdown>
            </tag-input>
```

```
public validators = [
        (control: any): any =>
        {
            if (!this._formattingService.IsValidEmail(<string>control.value))
            {
                return { invalidEmail: true };
            }
            return null;
        }
    ];

    public errorMessages = {
        'invalidEmail': 'Invalid email',
    };
```

**Comment by clement911** [comment]
When the user select one of the autocomplete items, the `control` parameter passed to the validation function has a `value` set to an empty string, causing it to fail validation.

**Comment by clement911** [comment]
That seems like a bug right?

**Comment by Gbuomprisco** [comment]
Hi @clement911, this should be fixed in v 1.8.2.

**Comment by clement911** [comment]
This works now thanks!
